crosswalks-sssom/template/TARGET_VOCAB-codemeta-mappings.csv

```diff
@@ -1,17 +1,78 @@
-source_term,codemeta_term,type_relation,combined_mappings,comments
-repository,codeRepository,exact_match,,
-version,softwareVersion,exact_match,,
-file,downloadUrl,exact_match,,
-version,softwareVersion,exact_match,,
-note,releaseNotes,exact_match,,
-author,author,exact_match,,
-year,datePublished,part_of,$(year)+$(month), 
-month,datePublished,part_of,$(year)+$(month),
-editor,editor,exact_match,,
-license,license,exact_match,,
-publisher,publisher,exact_match,,
-abstract,description,exact_match,,
-doi,identifier,more_specific_than,,
-hal_id,identifier,more_specific_than,,
-url,url,exact_match,,
-organization,affiliation,exact_match,,
+source_term,codemeta_term,combined_mappings,comments
+,applicationCategory,,
+,applicationSubCategory,,
+,author/affiliation/identifier,,
+,author/affiliation/name,,
+,author/email,,
+,author/familyName,,
+,author/givenName,,
+,author/identifier,,
+,author/roleName,,

```

Comment:
Not sure if this can be mapped as is, but I'm merging anyway and we will discuss this in https://github.com/codemeta/codemeta/issues/397